scripts/deploy-v2.0.9.js

```diff
@@ -205,7 +205,11 @@ async function main() {
 
         //add allowed tokens
         console.log(`Adding Allowed Tokens...`);
-        network.registry.addAllowedTokens([networkToken.contractAddress, rewardToken.contractAddress, bountyTransactional.contractAddress]);
+        // Reward Tokens
+        network.registry.addAllowedTokens([rewardToken.contractAddress]);
```

Comment:
am thinking if its not a good idea to _also_ add the reward token to the arguments ? @clarkjoao @hvasconcelos